third_party/data-layer-helper/LICENSE

```diff
@@ -0,0 +1,14 @@
+Copyright 2012 Google Inc. All Rights Reserved.
```

Comment:
Let's see if we can do without this. If we do still need it - we will need to submit it in a separate PR.